operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -853,7 +853,7 @@ def customizeNetworkAccessPoint(self, nap, listen_address):
     # FWIW there's theoretically no need to 'add' or 'replace' when empty
     #   since the runtime default is the server listen-address.
     nap_name=nap.getName()
-    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0:
+    if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0 and not (nap_name.startswith('istio-')) and not (nap_name == 'T3Channel'):
```

Comment:
Checking for the specific nap name 'T3Channel' looks iffy - I think this is the name we use in our sample and suspect it should be keyed keyed off of the ISTIO_ENABLED flag - something like:

`if not (nap.getListenAddress() is None) and len(nap.getListenAddress()) > 0 and not (nap_name.startswith('istio-')) and ((self.env.ISTIO_ENABLED == 'true') or not (nap_name == 'T3Channel')):`

assuming that `self.ISTIO_ENABLED = self.getEnvOrDef('ISTIO_ENABLED', 'false')` has been added in the global var init section at the top of this script